python/cuml/test/test_fil.py

```diff
@@ -465,41 +445,41 @@ def test_lightgbm(tmp_path, num_classes):
                          random_state=43210,
                          classification=True)
     train_data = lgb.Dataset(X, label=y)
+    num_round = 5
+    model_path = str(os.path.join(tmp_path, 'lgb.model'))
 
     if num_classes == 2:
         param = {'objective': 'binary',
                  'metric': 'binary_logloss',
                  'num_class': 1}
-    else:
-        param = {'objective': 'ova',  # 'multiclass', would use softmax
-                 'metric': 'multi_logloss',
-                 'num_class': num_classes}
-    num_round = 5
-    model_path = str(os.path.join(tmp_path, 'lgb.model'))
-
-    bst = lgb.train(param, train_data, num_round)
-    bst.save_model(model_path)
-    if num_classes == 2:
-        # binary classification
-        gbm_proba = bst.predict(X)
+        bst = lgb.train(param, train_data, num_round)
+        bst.save_model(model_path)
         fm = ForestInference.load(model_path,
                                   algo='TREE_REORG',
                                   output_class=True,
                                   model_type="lightgbm")
-        fil_proba = fm.predict_proba(X)
-        assert np.allclose(gbm_proba, fil_proba[:, 1], 1e-2)
+        # binary classification
+        gbm_proba = bst.predict(X)
+        fil_proba = fm.predict_proba(X)[:, 1]
         gbm_preds = (gbm_proba > 0.5)
         fil_preds = fm.predict(X)
         assert array_equal(gbm_preds, fil_preds)
+        np.testing.assert_allclose(gbm_proba, fil_proba,
+                                   atol=proba_atol[num_classes > 2])
     else:
         # multi-class classification
-        # FIL doesn't yet support predict_proba() for multi-class
-        # TODO: Add a test for predict_proba() when it's supported
-        gbm_preds = bst.predict(X)
-        gbm_preds = gbm_preds.argmax(axis=1)
```

Comment:
since lightgbm doesn't support probabilities without the sklearn API, using it here for both predictions